docs/features/configuration.md

```diff
@@ -75,17 +75,19 @@ See [Docker environment variables](https://docs.docker.com/engine/reference/comm
 
 3. Read the Go context for the **DOCKER_HOST** key. E.g. `ctx.Value("DOCKER_HOST")`. This is used internally for the library to pass the Docker host to the resource reaper.
 
-4. Read the default Docker socket path, without the unix schema. E.g. `/var/run/docker.sock`
+4. Read the host endpoint for the current Docker context in the Docker configuration file. E.g. `~/.docker/config.json`.
 
-5. Read the **docker.host** property in the `~/.testcontainers.properties` file. E.g. `docker.host=tcp://my.docker.host:1234`
+5. Read the default Docker socket path, without the unix schema. E.g. `/var/run/docker.sock`
```

Comment:
suggestion: I think we need to clarify this as I'm not sure I understand where it is getting the socket from?